rotkehlchen/tests/unit/test_bitcoin.py

```diff
@@ -350,3 +355,36 @@ def test_valid_bitcoin_chains():
     for blockchain in SupportedBlockchain:
         if blockchain not in (SupportedBlockchain.BITCOIN, SupportedBlockchain.BITCOIN_CASH):
             assert blockchain in NON_BITCOIN_CHAINS
+
+
+def test_bitcoin_balance_api_resolver():
+    # Test blockstream will be hit for bch
+    addresses = [
+        BTCAddress('3FZbgi29cpjq2GjdwV8eyHuJJnkLtktZc5'),
+        BTCAddress('34SjMcbLquZ7HmFmQiAHqEHY4mBEbvGeVL'),
+        BTCAddress('3J7sT2fbDaF3XrjpWM5GsUyaDr7i7psi88'),
+        BTCAddress('36Z62MQfJHF11DWqMMzc3rqLiDFGiVF8CB'),
+        BTCAddress('33k4CdyQJFwXQD9giSKyo36mTvE9Y6C9cP'),
+    ]
+    expected_balance = {
+        '34SjMcbLquZ7HmFmQiAHqEHY4mBEbvGeVL': FVal(0E-8),
```

Comment:
Use the zero constant here and below
```suggestion
        '34SjMcbLquZ7HmFmQiAHqEHY4mBEbvGeVL': ZERO,
```